src/dms-preview/azext_dms/custom.py

```diff
@@ -332,14 +350,13 @@ def extension_handles_scenario(
         source_platform,
         target_platform,
         task_type=""):
-    # Add scenario types to this list when moving them out of this extension (preview) and into the core CLI (GA)
+    # Remove scenario types from this list when moving them out of this extension (preview) and into the core CLI (GA)
     ExtensionScenarioTypes = [
         ScenarioType.sql_sqldb_online,
         ScenarioType.mysql_azuremysql_online,
         ScenarioType.postgres_azurepostgres_online,
         ScenarioType.mongo_mongo_offline,
         ScenarioType.mongo_mongo_online,
-        ScenarioType.oracle_azurepostgres_offline,
```

Comment:
Is there any command will be affected by the change here?

Reply:
No. That combination is not valid. This was a fix so that it would fail validation and not allow it.